### DISCUSSION TITLE
Error loading MP3 files

### DISCUSSION THREAD

**Discussion by MrDimain** [TortugaPower/BookPlayer#877] [General]
Greetings!
I faced such problem. BookPlayer does not play some files. The number of seconds is set to zero, although the file is played separately.
I am attaching an example MP3 file.

(Note: File removed)

**Comment by pichfl** [comment]
Try [mp3val](https://mp3val.sourceforge.net) to verify your files are intact. Your sample is not.

```sh

❯ mp3val audio.mp3
Analyzing file "audio.mp3"...
WARNING: "audio.mp3": Wrong CRC in 1 frames
WARNING: "audio.mp3": Wrong number of MPEG frames specified in Xing header (45263 instead of 45260)
WARNING: "audio.mp3": Wrong number of MPEG data bytes specified in Xing header (0 instead of 18916834)
INFO: "audio.mp3": 45260 MPEG frames (MPEG 1 Layer III), +ID3v1+ID3v2, CBR, CRC
```

The critical part is the `MPEG data bytes specified in Xing header`, which explains the 0 you experience.
